config/routes.rb

```diff
@@ -14,6 +14,8 @@
 
     get "/user", to: "user#show"
 
+    resources :oidc_users, only: %i[update], param: :subject_identifier, path: "oidc-users"
```

Comment:
Is the path specified here to have `/api/oidc-users` instead of `/api/oidc_users`?

Reply:
Yes

Reply:
What's the advantage of using `resources` if we only need `update`?

Reply:
I can imagine us wanting a `delete` endpoint in the future too.  But even if we don't, using `resources` seems like a good way to keep in mind the principle that controllers should only have the CRUD actions where possible.